docs/release/trg-1/trg-1-2.md

```diff
@@ -14,33 +14,28 @@ If a repository contains code which needs to be configured/setup or installed, t
 
 If these instructions are too long, they can be moved into a dedicated `INSTALL.md` file.
 
-Having a proper "install.md" file is crucial for ensuring a seamless and user-friendly experience for the product-users.
+Having a proper `INSTALL.md` file is crucial for ensuring a seamless and user-friendly experience for the product-users.
 
-In conclusion, a proper install.md file tailored for end-users, but also developers. It is not a technical necessity, it's a strategic investment in user satisfaction, community engagement, and the overall success of the product. It reflects a commitment to inclusivity and usability, ensuring that a diverse user base can effortlessly install and enjoy the benefits of your product.
+In conclusion, a proper `INSTALL.md` file should be mainly adapted to end-users, but also developers. It is not a technical necessity.
+It's a strategic investment in user satisfaction, community engagement, and the overall success of the product. It reflects a commitment to inclusivity and usability, ensuring that a diverse user base can effortlessly install and enjoy the benefits of your product.
 
 ## Description
 
 Create a `INSTALL.md` file in the main folder of the repository and add all installation relevant information.
 
 ## Best Practices
 
-- add a clear installing guide through our official helm chart repo
-- add table of content of your installation guide
-- add a numeric step-by-step installation guide with command copy options
-- add prerequisites section for the installation
-- add a reference your helm chart README.md file
-- add guidance on how to verify the installation
-- add local deployment advisories
-- add a troubleshooting section
-- add a FAQ section
-- add additional resources information
-- add uninstallation guide
-- add upgrade guide
-- add security considerations
-- add contact information for support or feedback
-- add a thankful note for the user for using the product
-
-## Example Installation Guide
+- a clear installing guide through our official helm chart repo
+- table of content of your installation guide
+- prerequisites section for the installation
+- a numeric step-by-step installation guide with command copy options
+- a reference your helm chart `README.md` file
+- install verification / uninstallation / upgrade guide if needed local deployment advisories
+- a FAQ and troubleshooting section
+- additional resources information
+- contact information for support or feedback
```

Comment:
Is this even necessary? We use GH isses anyways by default
```suggestion
```

Reply:
Jep we so, but maybe other Teams follow an other strategy and could link more information for support